fuel-vm/src/predicate.rs

```diff
@@ -425,6 +460,43 @@ mod tests {
                     PanicReason::PredicateReturnedNonOne,
                 )),
             ),
+            (
+                // Use `LDC` with mode `2` to load the part of the predicate from the
+                // transaction.
+                vec![
+                    // Skip the return opcodes. One of two opcodes is a good opcode that
```

Comment:
All the comments are based on the case where it succeed and not adapted for this case of failing

Reply:
Fixed=)